src/pages/contact.js

```
@@ -1,11 +1,32 @@
 import React from "react"
+import { graphql } from "gatsby"
 import Layout from "../components/layout"
 
-const ContactPage = () => {
+const ContactPage = ({ data }) => {
+  // Get the consultation link from Contentful
+  const consultationLink = data?.allContentfulHomepageLink?.nodes?.find(
+    link => link.text && link.text.toLowerCase().includes('consultation')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)